target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -530,4 +548,76 @@ contract EntropyTest is Test {
         assertEq(providerInfo1.uri, provider1Uri);
         assertEq(providerInfo1.commitmentMetadata, provider1CommitmentMetadata);
     }
+
+    function testSetAdminByAdmin() public {
```

Comment:
consider making a separate test file for this stuff like `EntropyAdminTest`. These tests don't depend on the same setup as the existing tests in this file, so two files will be easier for future readers to understand.